Tests/KlaviyoSwiftTests/__Snapshots__/KlaviyoAPITests/testSuccessfulResponseWithEvent.1.txt

```diff
@@ -9,7 +9,10 @@
   ▿ httpMethod: Optional<String>
     - some: "POST"
   ▿ allHTTPHeaderFields: Optional<Dictionary<String, String>>
-    - some: 0 key/value pairs
+    ▿ some: 1 key/value pair
+      ▿ (2 elements)
+        - key: "X-Klaviyo-Retry-Attempt"
+        - value: "0/50"
```

Comment:
When an actual device sends, the first send will be 1/50, ya?
I had to do this in my tests too, just want to confirm.

Reply:
Yeah I tested this using a proxy, it starts at 1. I will update the tests.